### PR TITLE
Added Corporate Advisory Banner Component

### PR DESCRIPTION
**Pull request by juliemtaylor**

### CODE REVIEW

File: new-components/corporate-homepage-advisory.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    <style>
+      .v2 .alert-bar-container {
+        background-color: #02132D;
+        border-top: 1px solid #7c7c7c;
+
+        text-align: center;
+        vertical-align: middle;
+      }
+      .v2 .alert-bar-container .container_12 {
+        display: table;
```
Comment (orderedlist) [inline new-components/corporate-homepage-advisory.html]: I might be a little nervous about adjusting the styles of the grid system, even though it's specified inside the `.alert-bar-container`. Perhaps we add another div to wrap the children here to add the `display: table;`, but I'll let you make that call.

File: new-components/corporate-homepage-advisory.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        min-height: 60px;
+      }
+      .v2 .alert-bar-label {
+        color: #fff;
+      }
+      .v2 .alert-bar-text {
+        color: #87CDFF;
+        display: table-cell;
+        font-weight: 400;
+        font-size: 14px;
```
Comment (orderedlist) [inline new-components/corporate-homepage-advisory.html]: I know a lot of the other one sizes in the specs/components are sized in `em`. Not sure if it matters here, but `1.4em` would be roughly the same and be consistent.

File: new-components/corporate-homepage-advisory.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      <!-- start of new component -->
+      <div class="alert-bar-container">
+        <div class="container_12"><!-- container_12 class should already be included in 960 grid css -->
+
+          <!-- add dynamtic link to specific advisory item -->
+          <a class="alert-bar-text" href="#">
+            <span class="alert-bar-label">Advisory:</span> Demonstrations in French Guiana - Bulletin 1
+          </a>
+
+          <!-- add dynamtic link to all advisory items -->
```
Comment (orderedlist) [inline new-components/corporate-homepage-advisory.html]: Love these comments. 😍

File: new-components/corporate-homepage-advisory.html (hunk 1 above)
Comment (juliemtaylor) [inline new-components/corporate-homepage-advisory.html]: You're right about this. How do you feel about .v2 .alert-bar-container>div? Or, I can add a new class to the inner wrapper.

Comment (orderedlist) [inline new-components/corporate-homepage-advisory.html]: I'd probably add a class to be declarative.

Comment (juliemtaylor) [inline new-components/corporate-homepage-advisory.html]: I think I've got it. :)

File: new-components/corporate-homepage-advisory.html (hunk 3 above)
Comment (juliemtaylor) [inline new-components/corporate-homepage-advisory.html]: 👍

**Pull request merged**